modules/ROOT/pages/tutorial-avx2-scheduling.adoc

```diff
@@ -0,0 +1,591 @@
+
+Use the following steps to label Kubernetes nodes that support AVX2 by using a custom DaemonSet:
+
+. <<#define-avx2-label, Define the AVX2 node label>>
+. <<#create-daemonset-manifest, Create the DaemonSet manifest>>
+. <<#deploy-daemonset, Deploy the DaemonSet>>
+. <<#verify-node-labels, Verify node labels>>
+
+[#define-avx2-label]
+==== Define the AVX2 Node Label
+
+Define the AVX2 node label to identify nodes that support the AVX2 CPU extension.
+
+[source]
+----
+cpu.feature/AVX2=true
+----
+
+[#create-daemonset-manifest]
+==== Create the DaemonSet Manifest
+
+Create a DaemonSet manifest named `avx2-node-labeler.yaml` with the following content that detects AVX2 support and applies the node label.
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: avx2-labeler-sa
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: avx2-labeler-role
+rules:
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get", "patch", "update"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: avx2-labeler-binding
+subjects:
+- kind: ServiceAccount
+  name: avx2-labeler-sa
+  namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: avx2-labeler-role
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: avx2-node-labeler
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: avx2-node-labeler
+  template:
+    metadata:
+      labels:
+        app: avx2-node-labeler
+    spec:
+      serviceAccountName: avx2-labeler-sa
+      containers:
+      - name: labeler
+        image: bitnami/kubectl:latest
+        command:
+        - /bin/bash
+        - -c
+        - |
+          if grep -qi "avx2" /host/proc/cpuinfo; then
+            kubectl label node "$NODE_NAME" cpu.feature/AVX2=true --overwrite
+          fi
+          sleep infinity
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        volumeMounts:
+        - name: host-proc
+          mountPath: /host/proc
+          readOnly: true
+      volumes:
+      - name: host-proc
+        hostPath:
+          path: /proc
+----
+
+[#deploy-daemonset]
+==== Deploy the DaemonSet
+
+Deploy the DaemonSet to run the AVX2 detection process on all nodes. 
+
+[source,console]
+----
+kubectl apply -f avx2-node-labeler.yaml
+----
+
+[#verify-node-labels]
+==== Verify Node Labels
+
+Verify that Kubernetes correctly applies the AVX2 label to supported nodes.
+
+[source,console]
+----
+kubectl get nodes -L cpu.feature/AVX2
+----
+
+[#pod-scheduling-with-nodeaffinity]
+== Pod Scheduling by Using nodeAffinity
+
+After you label nodes, configure the CouchbaseCluster resource to restrict pod scheduling to AVX2-capable nodes in one of the following ways:
+
+* <<#enforce-avx2-scheduling, *Enforce AVX2 Scheduling*>>: Recommended
```

Comment:
These bullets each need a full stop
(it's only on one to three word list items that we don't use one, afaik).